.github/workflows/ci.yml

```
@@ -10,7 +10,11 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node: [16, 17, 18, 19, 20, 22]
+        node:
+          - 18
```

If you want to keep `16`, you could add this suggestion
```suggestion
          - 16
          - 18
```